.github/actions/e2e/action.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: Test E2E
+description: Run e2e tests
+inputs:
+  working_directory:
+    description: "Working directory"
+    required: false
+    default: .
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        repository: neptune-ai/neptune-tensorboard
+        path: ${{ inputs.working_directory }}
+
+    - name: Install dependencies
+      working-directory: ${{ inputs.working_directory }}
+      run: pip install -e .[dev]
+      shell: bash
+
+    - name: Setup Graphviz
+      uses: ts-graphviz/setup-graphviz@v1
+      with:
+        ubuntu-skip-apt-update: true
+        macos-skip-brew-update: true
+
+    - name: Run tests
+      working-directory: ${{ inputs.working_directory }}
+      run: pytest -n auto -v
```

Comment:
Run tests in seperate process so that we patch individual copy of tensorboard for each test.